src/libs/actions/Report.js

```diff
@@ -753,15 +753,26 @@ function fetchActions(reportID, offset) {
 }
 
 /**
- * Get the actions of a report
+ * Get the actions of a report during pagination
  *
  * @param {Number} reportID
  * @param {Number} [offset]
  */
-function fetchActionsWithLoadingState(reportID, offset) {
-    Onyx.set(ONYXKEYS.IS_LOADING_REPORT_ACTIONS, true);
+function fetchActionsWithPagination(reportID, offset) {
```

Comment:
NAB, these names are a little misaligned with our current API refactor (that is maybe not very transparent) - but just mentioning it so @sketchydroide can be aware (and this will probably give them conflicts in the open PR - but no big deal).

Reply:
I think I'm actually removing this method completly in my PR, so I would not change this name method, or it will harder to merge my changes after